.github/workflows/ci.yaml

```diff
@@ -117,3 +140,28 @@ jobs:
         COVERALLS_REPO_TOKEN: 4nctuKHTnVtbdmty2FgdO3iiWm1nb7xc4
         COVERALLS_FLAG_NAME: python_${{ matrix.python-version }}-twisted_${{ matrix.twisted-version }}
         COVERALLS_PARALLEL: true
+
+  # Helper so that on GitHub repo settings we can configure to single job.
+  # Then required jobs can be updated directly form the code,
+  # without having to go the GitHub repo setting -> Protected branch
+  # and all the clicking.
+  treq-required:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    if: always()
```

Comment:
The current alwasy is ok for this project.

just a comment. 
Another option is not to run this when a job is cancelled, as a cancelled job will fail anyway.

Since we don't pay any money for this job, is not a big problem.

```suggestion
    if: ${{ !cancelled() }}
```